frameworks/helloworld/src/main/dist/examples/overlay.yml

```diff
@@ -99,47 +100,37 @@ pods:
     networks:
       dcos:
     tasks:
-      get-overlay:
-        goal: RUNNING
-        cpus: {{HELLO_CPUS}}
-        memory: {{HELLO_MEM}}
-        cmd: "./bootstrap && curl -X GET hello-overlay-0-server.hello-world.autoip.dcos.thisdcos.directory:1026 >> output && sleep $SLEEP_DURATION"
-        env:
-          SLEEP_DURATION: {{SLEEP_DURATION}}
-      get-overlay-vip:
-        goal: RUNNING
-        cpus: {{HELLO_CPUS}}
-        memory: {{HELLO_MEM}}
-        cmd: "./bootstrap && curl -X GET overlay-vip.hello-world.l4lb.thisdcos.directory:80 >> output && sleep $SLEEP_DURATION"
-        env:
-          SLEEP_DURATION: {{SLEEP_DURATION}}
-      get-host-vip:
+      check-comm:
         goal: RUNNING
         cpus: {{HELLO_CPUS}}
         memory: {{HELLO_MEM}}
-        cmd: "./bootstrap && curl -X GET host-vip.hello-world.l4lb.thisdcos.directory:80 >> output && sleep $SLEEP_DURATION"
-        env:
-          SLEEP_DURATION: {{SLEEP_DURATION}}
-      get-host:
-        goal: RUNNING
-        cpus: {{HELLO_CPUS}}
-        memory: {{HELLO_MEM}}
-        cmd: "./bootstrap && curl -X GET hello-host-0-server.hello-world.autoip.dcos.thisdcos.directory:4044 >> output && sleep $SLEEP_DURATION"
+        cmd: "echo hello && sleep $SLEEP_DURATION"
         env:
           SLEEP_DURATION: {{SLEEP_DURATION}}
+        health-check:
```

Comment:
We spoke offline.  Let's make this the `cmd` itself with `curl -f` and a goal state of FINISHED.  Goal of the exercise being that `deploy` plan COMPLETE == success.